### ISSUE TITLE
select item that is closest to the cursor in acessibility help dialog keybinding quickpick 

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#213417]
Suggested by @jooyoungseo, this would make it easier for users so they don't have to sort through the quick pick.

**Comment by meganrogge** [comment]
I just tried to implement this, and the limitation is we cannot currently get the command title from the `commandId`

**Comment by meganrogge** [comment]
Closing as I don't see a way to do this.

**Comment by meganrogge** [comment]
Reopening because would be much better if we could support this

**Comment by meganrogge** [comment]
use `MenuRegistry` to get the title from the command ID
